index.html

```diff
@@ -36,16 +37,19 @@
     </div>
     <section id="merchandise">
       <h2>Merchandise</h2>
-      <div class="merchphoto">
-        <a href="http://bfy.tw/AgFP"><img src="assets/shirt.jpg" alt=""></a>
-        <a href="http://bfy.tw/AgFR"><img src="assets/cutter.jpeg" alt=""></a>
-        <a href="http://bfy.tw/AgFS"><img src="assets/frosted-cookie.jpg" alt=""></a>
-      </div>
+      <!-- <div class="merchphoto"> -->
```

Comment:
You need to remove this and all other comments from your code (I can't add comments above this line so...yeah...)